indico/util/i18n.py

```diff
@@ -193,3 +221,41 @@ def extract_node(node, keywords, commentTags, options, parents=[None]):
         for cnode in ast.iter_child_nodes(node):
             for rslt in extract_node(cnode, keywords, commentTags, options, parents=(parents + [node])):
                 yield rslt
+
+
+def po_to_json(po_file, locale=None, domain=None):
+    """
+    Converts *.po file to a json-like data structure
+    """
+    with open(po_file, 'rb') as f:
+        po_data = read_po(f, locale=locale, domain=domain)
+
+    messages = dict((message.id[0], message.string) if message.pluralizable else (message.id, [message.string])
+                    for message in po_data)
+
+    messages[''] = {
+        'domain': po_data.domain,
+        'lang': str(po_data.locale),
+        'plural_forms': po_data.plural_forms
+    }
+
+    return {
+        (po_data.domain or ''): messages
+    }
+
+
+@after_process.connect
```

Comment:
**wrong signal** - `after_process` runs after every single request! you want `app_created` instead.